module/documents/item.mjs

```diff
@@ -462,7 +462,7 @@ export default class Item5e extends SystemDocumentMixin(Item) {
     const requireEquipped = (this.type !== "consumable")
       || ["rod", "trinket", "wand"].includes(this.system.type.value);
     if ( requireEquipped && (this.system.equipped === false) ) return true;
-    return this.system.attunement === CONFIG.DND5E.attunementTypes.REQUIRED;
+    return this.system.attuned || (this.system.attunement !== "required");
```

Comment:
I think this needs to be the inverse since we're returning whether the effect is suppressed or not:

```suggestion
    return !this.system.attuned && (this.system.attunement === "required");
```